fluentd/Dockerfile

```diff
@@ -76,6 +76,4 @@ RUN mkdir -p /var/log/journal
 # to prevent a race condition
 COPY include/wait-for-istio-proxy.rb /wait-for-istio-proxy.rb
 
-USER fluent
```

Comment:
Is it necessary to remove this? This was just added like 12 days ago.

Reply:
Followed up offline:  it does seem necessary.

Reply:
Link to platform-pvt discussion about this, agreeing to revert. https://astronomer.slack.com/archives/C021EE4HNQJ/p1629146778009900